pkg/gitlab/issues.go

```diff
@@ -0,0 +1,52 @@
+package gitlab
+
+import (
+	"context"
+	"time"
+
+	"github.com/grafana/gitlab-datasource/pkg/models"
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+	"github.com/xanzy/go-gitlab"
+)
+
+// Issues is a slice of git issues
+type Issues []*gitlab.Issue
+
+// Frames converts the list of issues to a Grafana DataFrame
+func (c Issues) Frames() data.Frames {
+	frame := data.NewFrame(
+		"issues",
+		data.NewField("id", nil, []string{}),
```

Comment:
```suggestion
		data.NewField("id", nil, []int64{}),
```

Note that the [`Issue.ID`](https://godoc.org/github.com/xanzy/go-gitlab#Issue) field is typed as an `int`.
There's no support for `int` in `data.Frames`, so we need to choose among (int8, int16...) so I'd go for `int64` because as this field represents an identifier, I'd expect huge values.